.travis.yml

```diff
@@ -24,12 +24,11 @@ env:
 
 matrix:
   include:
-    - env: CONFIG=linux_noarch UPLOAD_PACKAGES=True DOCKER_IMAGE=condaforge/linux-anvil-ppc64le
-      os: linux-ppc64le
-      arch: ppc64le
-      language: generic
+      - env: CONFIG=linux_noarch UPLOAD_PACKAGES=True DOCKER_IMAGE=condaforge/linux-anvil-ppc64le
+        os: linux-ppc64le
+        arch: ppc64le
+        language: generic
 
-script:
-  -  cd conda-recipes/tensor2tensor-feedstock
-  -  "travis_wait 60 sleep 3600 &"
-  -  ./ci_support/run_docker_build.sh
+        script:
```

Comment:
I think indentation is significant in YAML. Does it matter that this `script:` stanza has been indented here?

Reply:
Just copied it but yes I think it has more than required indentation. I'll just correct it.